project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clj-http "3.0.0-SNAPSHOT"
+(defproject lambdakazan/clj-http "2.0.4-SNAPSHOT"
```

Comment:
Sorry, I thought pull request will compare with a commit, not with a branch. Should I make another PR?